test/controllers/PenaltyDetailsControllerSpec.scala

```diff
@@ -81,20 +81,6 @@ class PenaltyDetailsControllerSpec extends SpecBase with MockPenaltyDetailsServi
       }
     }
 
-    "an authenticated user requests details for an unsupported tax regime" should {
```

Comment:
Can you put this back; this is still a valid test and not specific to income tax - it was just using the IT regime key as an example of an invalid String. You can substitute this with any made up value.